autoload/go/jobcontrol.vim

```diff
@@ -99,9 +99,10 @@ function! s:on_exit(job_id, exit_status) abort
 
   call s:callback_handlers_on_exit(s:jobs[a:job_id], a:exit_status, std_combined)
 
+  let l:listtype = go#list#Type("locationlist")
```

Comment:
Can you change this to `quickfix` as well? I know it was already like this before, but let us be consistent with the others as well. It also fixes https://github.com/fatih/vim-go/issues/1019 as well :)